master/reference/advanced/etcd-rbac/kubernetes-advanced.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Kubernetes set ups utilizing etcd RBAC
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes advanced configuration
+options of individual component that can be configured and offers some help
```

Comment:
Question, is the only real difference between this and the advanced section about limiting the Calico component access?  I think that is the case, but it's not clear.